lib/services/services_private.h

```diff
@@ -42,6 +42,10 @@ struct svc_action_private_s {
     DBusPendingCall* pending;
     unsigned timerid;
 #endif
+
+#if SUPPORT_SYSTEMD
+    char *job_path;
```

Comment:
This commit is missing the changes in `services_action_free()` and  `process_unit_method_reply()` from e61ce94. They get added in the next commit but belong in this one.

Reply:
Bah, `git am --reject` gets me again.